snomed/com.b2international.snowowl.snomed.ql/src/com/b2international/snowowl/snomed/QL.xtext

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2019 B2i Healthcare Pte Ltd, http://b2i.sg
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+grammar com.b2international.snowowl.snomed.QL
+with com.b2international.snowowl.snomed.ecl.Ecl
+
+generate ql "http://www.b2international.com/snowowl/snomed/QL"
+
+Query:
+	{Query} (disjunction=Disjunction)?;
+	
+Disjunction:
```

Comment:
Supporting `OR`/`AND`/`MINUS` operators require handling of ambiguity issues, when you specify expressions like this: `A OR B AND C MINUS D`.
Please fix it by introducing recursion in the language via nested expressions.